File-Structure-React-ES6-Webpack-Project.md

```diff
@@ -0,0 +1,117 @@
+# File Structuring
```

Comment:
```diff
-# File Structuring
+# React with Webpack File Structure
```

Titles need to be descriptive, if it is React and webpak it should mention it otherwise it will show on the live wiki as just `File Structure`